test/fixtures/overrides/package.json

```diff
@@ -14,7 +14,8 @@
       }
     ],
     "rules": {
-      "import/no-extraneous-dependencies": 0
+      "import/no-extraneous-dependencies": 0,
+      "ava/no-ignored-test-files": 0
```

Comment:
I had to ignore this since this is a package inside of the fixtures folder anyways, and so it triggered. It probably wouldn't be worth it to fix ava/no-ignored-test-files just for this.